client/dist/styles/qA.css

```diff
@@ -2,9 +2,9 @@
 /**************** BEGINNING OF Q/A PORTION **************** */
 #q-container {
   min-width: 720px; /* For Dev */
-  padding: 0px 60px;
```

Comment:
Forgot, but i added a line to the CSS to bring all the components off the left side of the screen, centered in the middle